### PR TITLE
document new unresolved call syntax

### PR DESCRIPTION
**Pull request by naftali-g**
Naming convention:
 - PRs for features that are in design should have the "proposal" label
 - PRs for features that haven't landed yet should have the "future" label
 - PRs for upcoming releases should have the "release" label
 - PRs with new documentation for existing features should have the "existing feature" label

Before requesting review:
 - Make sure there is a ticket in the DOC board in Jira
 - Make sure CI is passing
   - Spell check failure may require adding backticks around code or updating `spelling_wordlist.txt`
   - See `README.md` for information about style and markdown syntax
   - If the CI Details link gives a 404, you need to log in to readthedocs.com
 - Add link to generated documentation here
   - you can find this by following the read the docs link from the CI check
 - Ask for help in #documentation

Jira ticket: TODO
Link to generated documentation: https://certora-certora-prover-documentation--287.com.readthedocs.build/en/287/docs/cvl/methods.html#catch-unresolved-calls-entry

### CODE REVIEW

**Review by shellygr — COMMENTED** [review]
lgtm. this is cool! leaving approval to Johannes

File: docs/cvl/methods.md [hunk 1]
```diff
@@ -195,31 +195,48 @@ shown in the web report can indicate whether a summary was applied.
...
+_not_ be applied to this unresolved call - only an entry that matches `D.bar`
+will be used.
 ```
 
-A catch unresolved-calls entry can only be summarized with a dispatch list
+Catch unresolved-calls entries can only be summarized with a dispatch list
 summary (and a dispatch list summary is only applicable for a catch
-unresolved-calls entry).
+unresolved-calls entries).
+
```
Comment (christiane-certora) [inline docs/cvl/methods.md:239]: Not introduced by you, but I was reading the generated doc so I didn't know which part was changed - a few lines below this there's some leftover merge indicators (257, 259, 262) that would be nice to clean up.

Comment (naftali-g) [reply docs/cvl/methods.md:239]: Good catch! I'll make a new PR for that though - this onecould only be merged once the feature is released to stable version, but those merge indicators are already out in the wild

File: docs/cvl/methods.md [hunk 2]
```diff
@@ -195,31 +195,48 @@ shown in the web report can indicate whether a summary was applied.
...
+Catch unresolved-calls entries are a special type of summary declaration that
 instructs the Prover to replace calls to unresolved external function calls
 with a specific kind of summary, dispatch list.
 By default, the Prover will use an {ref}`AUTO summary <auto-summary>` for
 unresolved function calls, but that may produce spurious counter examples.
-The catch unresolved-calls entry lets the user refine the summary used for
+Catch unresolved-calls entries let the user refine the summary used for
 unresolved function calls.
 
+One can specify the scope for which the unresolved summary will apply. The options are:
```
Comment (johspaeth) [inline docs/cvl/methods.md:220]: > One can specify the scope for

I think we can be slightly more precise here what we mean by "scope". The reader may question what you meant by scope, see this example:

```
unresolved external in C.foo() => DISPATCH [
        D.baz() 
    ] default HAVOC_ECF;
```

Scope could be interpreted as 1. `C.foo` or the list containing 2. `D.baz()`. 

Maybe...?

```suggestion
One can specify the scope (`unresolved external in <scope>`) for which the unresolved summary will apply.  The options are:
```

**Review by johspaeth — COMMENTED** [review]
Minor suggestion for improvement. What do you think?

**Review by johspaeth — APPROVED** [review]

**Pull request merged**